roles/openshift_repos/tasks/centos_repos.yml

```diff
@@ -19,6 +19,6 @@
     src: "{{ item }}"
     dest: "/etc/yum.repos.d/{{ (item | basename | splitext)[0] }}"
   with_first_found:
-    - "CentOS-OpenShift-Origin{{ ((openshift_version | default('')).split('.') | join(''))[0:2] }}.repo.j2"
+    - "CentOS-OpenShift-Origin{{ ((openshift_version | default('')).split('.') | join(''))[0:3] }}.repo.j2"
```

Comment:
That expression will fail with 3.9.0, it will look for 390.repo. 
I think an expression that will strip the x.y version would be
```((openshift_version | default('')).split('.',2)[0:2] | join(''))```

Reply:
Anyway, I see that today the 3.10 rpms got uploaded to the mirror - http://mirror.centos.org/centos/7/paas/x86_64/openshift-origin/
So this patch is not needed I think

Reply:
@rgolangh 
> That expression will fail with 3.9.0, it will look for 390.repo.

That is incorrect.

> So this patch is not needed I think

Also incorrect.

Reply:
I believe this would be the python equivalent (snippet is from ipython):
```
In [36]: "".join("3.9.0".split('.'))[0:3]
Out[36]: '390'

In [37]: "".join("3.10.0".split('.'))[0:3]
Out[37]: '310'
```
```
In [38]: "".join("3.9.0".split('.')[0:2])
Out[38]: '39'

In [39]: "".join("3.10.0".split('.')[0:2])
Out[39]: '310'
```

Reply:
master and release-3.10 branches cannot be used to install 3.9 anyway, due to the switch to static pods